src/views/bridge-details/bridge-details.view.tsx

```diff
@@ -91,6 +92,8 @@ const BridgeDetails: FC = () => {
     status: bridge.status === "successful" ? bridge.data.status : undefined,
   });
 
+  const getBridgeAbortController = useRef<AbortController>(new AbortController());
```

Comment:
`getBridgeAbortController` -> `bridgeAbortController`

Reply:
Why? this is the `AbortController` of the function `getBridge` so it's the `getBridgeAbortController`